vtp-pensjon-frontend/src/components/Snitch/RequestResponseList.tsx

```diff
@@ -50,15 +52,126 @@ export default function RequestResponseList(props: {
                 <thead>
                 <tr>
                     <th>Timestamp</th>
-                    <th>Method</th>
-                    <th>Path</th>
-                    <th>Status</th>
+                    <th>
+                    <Dropdown className="mx-2" autoClose="outside">
+                        <Dropdown.Toggle variant="secondary">Method</Dropdown.Toggle>
+                        <Dropdown.Menu>
+                            {props.requests
+                                .filter((request, i) => {
+                                    return props.requests.findIndex((value, i) => { return value.method === request.method }) === i
+                                })
+                                .map((request, i) => {
+                                return <Form className="mx-2">
+                                    <Form.Check
+                                        type={"checkbox"}
+                                        label={request.method}
+                                        onChange={() => {
+                                            var pathMethods = props.filters.methods
+                                            if (pathMethods.indexOf(request.method) !== -1) {
+                                                pathMethods = pathMethods.filter((method) => { return method !== request.method })
+                                            } else {
+                                                pathMethods = pathMethods.concat(request.method)
+                                            }
+                                            var updatedFilters = new RequestFilters()
+                                            updatedFilters.paths = props.filters.paths
+                                            updatedFilters.methods = pathMethods
+                                            updatedFilters.status = props.filters.status
+                                            props.setFilters(updatedFilters)
```

Comment:
Hadde vel vært best å trekke ut denne blokken og tilsvarende i funksjoner for å slippe noe særlig logikk i komponenten. 

Her kan du vel også bruke let fremfor var

Reply:
Agreed, fant ikke noen overordnet forbedring men fikk refraktorert det ned litt.